javalib/src/main/scala/java/lang/ClassLoader.scala

```diff
@@ -1,9 +1,15 @@
 package java.lang
 
+import scala.scalanative.native.stub
```

Comment:
We typically omit `scala.` from the imports as `scalanative` is already specific enough.